packages/requests/src/lib/requests-result.spec.ts

```diff
@@ -434,6 +443,96 @@ describe('requests result', () => {
     `);
   });
 
+  it('should cache response data', () => {
+    jest.useFakeTimers().setSystemTime(new Date('2020-01-01'));
+
+    const { state, config } = createState(withEntities<Todo>());
+
+    const store = new Store({ state, config, name: 'todos' });
+
+    const entities$ = store.pipe(selectAllEntities());
```

Comment:
Why do you need a store? You said that you want the data directly from the result

Reply:
I'm checking in the test if entites in the store still behave as expected when cacheResponseData is set